src/test/typescript-service-helpers.ts

```diff
@@ -1577,12 +1577,15 @@ export function describeTypeScriptService(TypeScriptServiceConstructor = TypeScr
 
 		beforeEach(<any> initializeTypeScriptService(TypeScriptServiceConstructor, new Map([
 			['file:///keywords-in-path/class/constructor/a.ts', 'export function a() {}'],
-			['file:///special-characters-in-path/%40foo/b.ts', 'export function b() {}']
+			['file:///special-characters-in-path/%40foo/b.ts', 'export function b() {}'],
+			['file:///windows/app/master.ts', '/// <reference path="..\\lib\\master.ts" />\nc();'],
+			['file:///windows/lib/master.ts', '/// <reference path="..\\lib\\slave.ts" />'],
+			['file:///windows/lib/slave.ts', 'function c() {}']
 		])));
 
 		afterEach(<any> shutdownTypeScriptService);
 
-		describe('getHovers()', <any> function (this: TestContext) {
+		describe('LSP server', <any> function (this: TestContext) {
```

Comment:
why this rename? I like to have this structure
- describe TypeScriptService
  - describe specific workspace context
    - describe method
       - it should ...